test/e2e/suite-ingress/suite-ingress-features/status.go

```diff
@@ -275,6 +275,20 @@ wrw7im4TNSAdwVX4Y1F4svJ2as5SJn5QYGAzXDixNuwzXYrpP9rzA2s=
 		assert.Contains(ginkgo.GinkgoT(), output, `status: "False"`, "status.conditions.status  is recorded")
 	})
 
+	ginkgo.It("check ApisixClusterConfig status is recorded", func() {
+		// create ApisixClusterConfig resource
+		clusterConfigName := "apisix-cluster-config"
```

Comment:
Hi, since you pass in "default" name to config, you can rmv this `clusterConfigName := "apisix-cluster-config"`

Reply:
May I know the reason why I should remove this? I didn't pass in any other value for the config name besides 'default'. Is there anything else that I'm missing?

Reply:
@afzal442 

I apologize for overlooking the fact that the sample on the official website specifies the use of the 'default' value as the default value for metadata.name.

Reply:
No issues @Chever-John. It's alright. I am also learning with you. 😄

Reply:
Thank you for your previous guidance, it helped me a lot. I was stuck creating a test environment before, but now I can work on this issue more efficiently:)